editor.js

```diff
@@ -82,7 +82,7 @@ function makeEditorShim(id) {
 
 
 function extension_editor_finish_run() {
-	if (typeof(unsafeWindow) !== 'undefined') {
+	if ((typeof(unsafeWindow) !== 'undefined') && (navigator.userAgent.indexOf('Mobile') === -1)) {
```

Comment:
@Wolvan, XKit.browser().mobile already exists, but it is implemented in such a way that it won't work here. This was to prevent mobile code from running on the desktop site and breaking everything.